flink-cdc-dist/pom.xml

```diff
@@ -62,4 +61,154 @@ under the License.
+                                    <type>jar</type>
+                                    <outputDirectory>${project.build.directory}/lib</outputDirectory>
+                                </artifactItem>
+
+                                <artifactItem>
+                                    <groupId>com.ververica</groupId>
+                                    <artifactId>flink-sql-connector-oceanbase-cdc</artifactId>
+                                    <version>${project.version}</version>
+                                    <destFileName>oceanbase-cdc-connector.jar</destFileName>
+                                    <type>jar</type>
+                                    <outputDirectory>${project.build.directory}/lib</outputDirectory>
+                                </artifactItem>
+
+                            </artifactItems>
+                        </configuration>
+                    </plugin>
+                    <plugin>
+                        <groupId>org.apache.maven.plugins</groupId>
+                        <artifactId>maven-assembly-plugin</artifactId>
+                        <executions>
+                            <execution>
+                                <id>dist</id>
+                                <goals>
+                                    <goal>single</goal>
+                                </goals>
+                                <phase>package</phase>
+                                <configuration>
+                                    <tarLongFileMode>gnu</tarLongFileMode>
+                                    <appendAssemblyId>false</appendAssemblyId>
+                                    <skipAssembly>false</skipAssembly>
+                                    <attach>false</attach>
+                                    <finalName>flink-cdc-connectors-${project.version}-bin</finalName>
+                                    <descriptors>
+                                        <descriptor>${basedir}/src/main/assembly/assembly.xml</descriptor>
+                                    </descriptors>
+                                </configuration>
+                            </execution>
+                        </executions>
+                    </plugin>
+                    <plugin>
+                        <artifactId>maven-antrun-plugin</artifactId>
+                        <executions>
+                            <execution>
+                                <id>dist</id>
+                                <goals>
+                                    <goal>run</goal>
+                                </goals>
+                                <phase>package</phase>
+                                <configuration>
+                                    <target>
+                                        <copy file="${project.build.directory}/flink-cdc-connectors-${project.version}-bin.tar.gz" tofile="${project.basedir}/../dist/flink-cdc-connectors-${project.version}-bin.tar.gz" overwrite="true" />
```

Comment:
Maybe we could skip the step of creating `dist` under the project root, and just leave the tar under `flink-cdc-dist/target`. The `dist` contains just one tarball, and we have to maintain a lot of things for it, like `.gitignore`, and one missing issue in your PR is that we need to remove it during `clean` phase.